server/src/main/java/org/candlepin/policy/js/compliance/StatusReasonMessageGenerator.java

```diff
@@ -89,6 +89,9 @@ else if (reason.getKey().equals("VCPU")) {
                 reason.getAttributes().get("covered"),
                 reason.getAttributes().get("has")));
         }
+        else if (reason.getKey().equals("UNMAPPEDGUEST")) {
+            reason.setMessage(i18n.tr("Has an unmapped guest subscription."));
```

Comment:
Maybe something like "Guest has not been reported on any host, using temporary unmapped guest subscription."